src/csharp/Grpc.Core/Server.cs

```diff
@@ -325,25 +344,41 @@ private void DisposeHandle()
             {
                 Logger.Warning(e, "Exception while handling RPC.");
             }
+
+            if (continuation != null)
```

Comment:
nit: from where I'm seeing `continuation` is only used for `AllowOneRpc` right? should it ever be null?